frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -58,6 +58,20 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
   const projectNumber = queryString.parse(query).projectNumber;
   const historyReplace = history.replace;
 
+  let exemptionRequest: boolean = formikRef?.current?.values.exemptionRequested;
+  let exemptionRationale: string = formikRef?.current?.values.exemptionRational;
+
+  const [rationaleRequired, setRationaleRequired] = useState(false);
+
+  exemptionRequest &&
```

Comment:
see comments in yup schema

Reply:
I'd prefer that this logic be moved to the ReviewProjectForm if possible, and then just use schema validation, rather then disabling the button. That way you provide the user feedback why that field is required, otherwise you are just disabling the button with no information as to why.

Reply:
Also, I'm trying to keep this file from getting any more complicated since a lot of critical logic executes here.

Reply:
Sounds good @devinleighsmith, I'll adjust